lib/translation-assistant.js

```diff
@@ -68,7 +68,7 @@ function stringToGroupKey(str) {
     var texts = maths.map(function (math) {
         var result = [];
 
-        allMatches(math, /\\text(?:bf)?{([^}]*)}/g, function (matches) {
+        allMatches(math, /\\text(?:bf)?\s*{([^}]*)}/g, function (matches) {
```

Comment:
Maybe we could extract this regex to a const and add it to `TEXT_REGEX` and `TEXTBF_REGEX` at the top of the file?

Reply:
Do we also want to accept newlines between the macro name and the parameters? Maybe we should use `[ \t]` instead?

Reply:
That makes sense, thanks. That would mean also changing `TEXT_REGEX` and `TEXTBF_REGEX` accordingly. Should I go ahead and do it?

Reply:
Dang - looks like the article/exercise editors actually accept one newline in there (and that's actually being used in three of our content strings) - so either `\s*` to keep it simple, or `[ \t]*\n?[ \t]*` to be precise.

Reply:
What about something like:

```
Good idea. What about something like:

```
        const regex = new RegExp(
            TEXT_REGEX.source + "|" + TEXTBF_REGEX.source, "g");

        allMatches(math, regex,
            (matches) => result.push(
                // TEXT_REGEX capture group is at index 1 and TEXTBF_REGEX at
                // index 2. One of the groups is expected to be `undefined`.
                matches[1] || matches[2]));
```

?

Reply:
@mpolyak thanks, I've pushed your solution, all tests passed.